### DISCUSSION TITLE
Best way to restart Pluto

### DISCUSSION THREAD

**Discussion by Kolaru** [fonsp/Pluto.jl#993] [Questions about using Pluto] [answered]
What is the best way to restart Pluto?

Currently I ctr-c the terminal I used to launch Pluto and restart it from there. I was wondering if there is a better way.

**Comment by fonsp** [comment, marked as answer]
Not that I know of!

**Comment by fonsp** [comment]
It is generally sufficient to restart the notebook process, without restarting Pluto itself: https://github.com/fonsp/Pluto.jl/wiki/%F0%9F%94%8E-Basic-Commands-in-Pluto#11-close-notebook
